tests/php/phpmessdetector_test_files/codesize_violation.php

```diff
@@ -0,0 +1,39 @@
+<?
+// Cyclomatic Complexity = 11
+class Foo {
+    public function example() {
+        if ($a == $b) {
+            if ($a1 == $b1) {
+                fiddle();
+            } elseif ($a2 == $b2) {
+                fiddle();
+            } else {
+                fiddle();
+            }
+        } elseif ($c == $d) {
+            while ($c == $d) {
+                fiddle();
+            }
+         } elseif ($e == $f) {
+            for ($n = 0; $n < $h; $n++) {
+                fiddle();
+            }
+        } else {
+            switch ($z) {
+                case 1:
+                    fiddle();
+                    break;
+                case 2:
+                    fiddle();
+                    break;
+                case 3:
+                    fiddle();
+                    break;
+                default:
+                    fiddle();
+                    break;
+            }
+        }
+    }
+}
+?>
```

Comment:
@SanketDG coala doesn't pick this up even when I run it individually on this file.
Command: `coala -V -b SpaceConsistencyBear -f s.php --flush-cache`
`s.php` is the following file without a newline at eof.